### DISCUSSION TITLE
sentence pair

### DISCUSSION THREAD

**Discussion by rajae-Bens** [aub-mind/arabert#42] [Q&A]
Hi,

I tired to apply this notebook AraBERT_Text_Classification_with_HF_Trainer_Pytorch_GPU.ipynb  for sentence pair

I tried to change this line to add text_b but I failed

train_dataset.add_examples(texts_or_text_and_labels=train_df['text'],labels=train_df['label'],overwrite_examples = True)

Could someone help plz

Thank u

**Comment by WissamAntoun** [comment]
Hey

the train_dataset is an instance of `SingleSentenceClassificationProcessor`, I think you need to write your own Dataset class.

You can take a look at the NER notebook which contains a dataset class
